frontend/src/features/admin-form/responses/IndividualResponsePage/PaymentSection.tsx

```diff
@@ -66,24 +36,19 @@ export const PaymentSection = ({
       ? { label: 'Disputed', colorScheme: 'warning' }
       : undefined // The remaining options should never appear.
 
+  const payoutTagProps =
+    payment.payoutId || payment.payoutDate
```

Comment:
Just curious but `payoutId` and `payoutDate` will always exists together right? i.e. 0 || 0 and 1 || 1?

Reply:
yup, this is a bit of a deficiency in the dto. but to avoid complication, i think this is fine. In principle, if any one of them is there, we should consider it successful (since a payout is already "known" to occur by ID or by date).